.gitignore

```diff
@@ -4,6 +4,8 @@ localData/
 dist/
 .yext
 .artifact-output
+.env
+!.env.local
```

Comment:
Is this line necessary?

Reply:
Yes, if we don't have it then .env.local gets ignored

Reply:
Hmmm I think we wanna ignore .env.local normally?

Reply:
We want .env.local to be checked in

Reply:
.env with !.env.local means that .env gets ignored, but .env.local does not get ignored which is what we want